docs/05-internals/development/vscode-config.md

```diff
@@ -0,0 +1,37 @@
+# Setup Visual Studio Code for development of DSP-API
+
+To have full functionality, the [Scala Metals](https://scalameta.org/metals/) plugin should be installed.
+
+Additionally, a number of plugins can be installed for convenience, but are not required. Those include but are by no means limited to:
+- Docker (to attach to running docker containers)
+- Stardog RDF grammar (.ttl sysntax highlighting)
+- Lua
+- REST client
```

Comment:
```suggestion
- Docker - to attach to running docker containers,
- Stardog RDF grammar - TTL syntax highlighting,
- Lua,
- REST client,
```

Reply:
I left out the comma at the end of the bulletpoint entries, but apart from that, I applied the suggestion